packages/shared/providers.yaml

```diff
@@ -6287,6 +6287,26 @@ shopify:
             suffix: .myshopify.com
             prefix: https://
 
+shopify-scim:
+    display_name: Shopify (SCIM API)
```

Comment:
Missing the connection docs link and the link to the specific step. See https://github.com/NangoHQ/nango/blob/master/packages/shared/providers.yaml#L235 and https://github.com/NangoHQ/nango/blob/master/packages/shared/providers.yaml#L244